test/unit/sso/openid_connect_test.rb

```diff
@@ -77,6 +77,20 @@ class OpenidConnectTest < ActiveSupport::TestCase
         assert_equal users(:one), subject.authenticated?
       end
     end
+
+    test "it accepts group paratameter in payload and authenticates the user" do
+      oidc_source = AuthSourceExternal.where(:name => 'External').first_or_create
+      external = FactoryBot.create(:external_usergroup, :auth_source => oidc_source)
+      usergroup = FactoryBot.create(:usergroup, :admin => true)
+      external.update(:usergroup => usergroup, :name => usergroup.name)
+
+      User.current = nil
+      payload['groups'] = usergroup.name
+      OidcJwt.any_instance.stubs(:decode).returns(payload.with_indifferent_access)
+      subject.authenticated?
```

Comment:
should we assert this worked? i.e.
```suggestion
      assert subject.authenticated?
```